Sources/PackageLoading/PackageExtensions.swift

```diff
@@ -227,6 +227,10 @@ extension Package {
             //TODO and then we should prefix all modules with their package probably
             //Suffix 'Tests' to test product so the module name of linux executable don't collide with
             //main package, if present.
+            // FIXME: Ignore C language test modules on linux for now.
+          #if os(Linux)
+            let testModules = testModules.filter { !($0 is ClangModule) }
```

Comment:
Let's print some kind of warning here, as opposed to just silently ignoring it. That will be really annoying for anyone using such a package on Linux, but that can be our incentive to fix it in a more sensible fashion.